index.html

```diff
@@ -37,7 +37,7 @@
     <h4 class="text-center mt-5">Handcrafted, home-made masterpieces</h4>
     <form class="form-inline mx-5" style="width:20rem ;">
       <input class="" type="search" placeholder="Enter your email address" aria-label="Search">
-      <button class="btn" type="submit">Get Started</button>
+      <button class="btn" type="submit">Let's Get Started</button>
```

Comment:
Please I suggest you kindly consider giving background color and paddings to your button so your design looks great